packages/tx/src/baseTransaction.ts

```diff
@@ -43,6 +47,8 @@ export abstract class BaseTransaction<TransactionObject> {
 
   public readonly common!: Common
 
+  public cache: TransactionCache
```

Comment:
Same thing here with the API: this can't be `public` since it is then exposed to the library users (which shouldn't be the case). I guess `protected` should be ok here though.

Small nit, since you need to do a small update anyhow: maybe do the initialization directly here as you have done for the block library? (unless there is some reason to here in tx do it in the constructor that I am not seeing right now).

Reply:
Updated. Not sure why I didn't follow the same approach as the block library, but fixed now